appveyor.yml

```diff
@@ -148,8 +148,8 @@ test_script:
   - sh src/ci/run.sh
 
 on_failure:
-  - cat %CD%\sccache.log
-  - cat C:\Users\appveyor\AppData\Local\Temp\1\build-cache-logs\*.log
+  - cat %CD%\sccache.log || exit 0
+  - cat C:\Users\appveyor\AppData\Local\Temp\1\build-cache-logs\*.log || exit 0
```

Comment:
In fact, I'll move it elsewhere and then we can just try it rather than merging.